### ISSUE TITLE
reversed order of colors on AnnoBar and DistToTSS

### ISSUE THREAD

**Issue by kzkedzierska** [GuangchuangYu/ChIPseeker#47]
Hi, 

I seem to be having an issue with those two plots. I tried looking at your code and coming up with an explanation but it was fruitless. When I generate the plots they have a different order: 

- for AnnoBar: at yours first is Promoter regions, then UTR and so on, and on mine, we start with Distal Intergenic
- and DistToTSS: at yours - 0-1kb, 1-3kb etc. while on mine >100kb, 10-100kb). 

The colors and labels match the values listed when accessing Genomic Summary etc. but especially DistToTSS doesn't make sense with the TSS in the middle and the order reversed.

examples (I tried on different data and different Rs)
https://drive.google.com/file/d/0B21lLuavNb9vY3h4Zm1UQ1pCZVE/view?usp=sharing
https://drive.google.com/file/d/0B21lLuavNb9vUWpYYjREU0FoM2M/view?usp=sharing

I can provide you with full sessionInfo if that will help, but to make it easier to read I only put section of it.
R version 3.3.1 (2016-06-21)
ggplot2_2.2.0, ggplot2_2.2.0, plyr_1.8.4, magrittr_1.5 

I would be grateful for any suggestions. 

Best,
Kasia

**Comment by GuangchuangYu** [comment]
pls follow <https://guangchuangyu.github.io/2016/07/how-to-bug-author/> and give me a reproducible example.

**Comment by kzkedzierska** [comment]
Ok, first of all sorry for my mistake, I should have prepared better. 

I now used the data from the package and got the same result. 

I follow the instruction in the link and checked for my version of the package and it's the latest. I produced two plots using data available with the package itself and got similar result. I attach the code, as well as the plots and sessionInfo. I'm not sure if that is what can be called of reproducible example, but I have no other idea at this point. 

```
require(ChIPseeker)
require(TxDb.Hsapiens.UCSC.hg19.knownGene)
txdb <- TxDb.Hsapiens.UCSC.hg19.knownGene
require(org.Hs.eg.db)

files <- getSampleFiles()
print(files)

peakAnno <- annotatePeak(files[[4]], tssRegion=c(-3000, 3000), TxDb=txdb, annoDb="org.Hs.eg.db")
plotAnnoBar(peakAnno)
plotDistToTSS(peakAnno, title="Distribution of transcription factor-binding loci\nrelative to TSS")
```

[featureDis](http://imageshack.com/a/img924/3854/jxC1X5.png)
[distToTSS](http://imageshack.com/a/img921/2227/LnScpr.png)

####Session Info
```
R version 3.3.1 (2016-06-21)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.9.5 (Mavericks)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] BiocInstaller_1.24.0                    org.Hs.eg.db_3.4.0                      TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [4] GenomicFeatures_1.26.0                  AnnotationDbi_1.36.0                    Biobase_2.34.0                         
 [7] GenomicRanges_1.26.1                    GenomeInfoDb_1.10.1                     IRanges_2.8.1                          
[10] S4Vectors_0.12.1                        BiocGenerics_0.20.0                     ChIPseeker_1.10.2                      

loaded via a namespace (and not attached):
 [1] SummarizedExperiment_1.4.0 qvalue_2.6.0               gtools_3.5.0               reshape2_1.4.2            
 [5] splines_3.3.1              colorspace_1.3-2           rtracklayer_1.34.1         XML_3.98-1.5              
 [9] DBI_0.5-1                  BiocParallel_1.8.1         RColorBrewer_1.1-2         plyr_1.8.4                
[13] stringr_1.1.0              zlibbioc_1.20.0            Biostrings_2.42.1          munsell_0.4.3             
[17] GOSemSim_2.0.2             gtable_0.2.0               caTools_1.17.1             memoise_1.0.0             
[21] labeling_0.3               UpSetR_1.3.1               biomaRt_2.30.0             Rcpp_0.12.8               
[25] KernSmooth_2.23-15         scales_0.4.1               gdata_2.17.0               plotrix_3.6-3             
[29] DO.db_2.9                  XVector_0.14.0             Rsamtools_1.26.1           gplots_3.0.1              
[33] gridExtra_2.2.1            ggplot2_2.2.0              digest_0.6.10              stringi_1.1.2             
[37] dplyr_0.5.0                grid_3.3.1                 DOSE_3.0.9                 tools_3.3.1               
[41] bitops_1.0-6               magrittr_1.5               lazyeval_0.2.0             RCurl_1.95-4.8            
[45] tibble_1.2                 RSQLite_1.1-1              GO.db_3.4.0                gridBase_0.4-7            
[49] assertthat_0.1             R6_2.2.0                   boot_1.3-18                GenomicAlignments_1.10.0  
[53] igraph_1.0.1   
```

**Comment by GuangchuangYu** [comment]
it's due to the change of ggplot2, see 'stacking bars' session in <https://blog.rstudio.org/2016/11/14/ggplot2-2-2-0/>.

This issue will be fixed in ChIPseeker >= 1.10.3.

Thanks for your feedback.
